bool a
---
int16 b
